src/main/resources/application-development.properties

```diff
@@ -1,12 +1,13 @@
 #Standard network settings
-server.port = 7777
+server.port=7777
 
-# H2
-spring.h2.console.enabled=true
-spring.h2.console.path=/h2
-
-# Datasource
-spring.datasource.url=jdbc:h2:file:~/habicus
-spring.datasource.username=admin
-spring.datasource.password=admin
-spring.datasource.driver-class-name=org.h2.Driver
+# MySQL Alternative For Development Persistence Store
+spring.datasource.url= jdbc:mysql://localhost:3306/habicus
+spring.datasource.username=root
+spring.datasource.password=${MYSQL_PASSWORD}
+spring.datasource.driver-class-name=com.mysql.jdbc.Driver
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5Dialect
+spring.datasource.platform=mysql
+spring.jpa.hibernate.ddl-auto=none
+spring.datasource.initialization-mode=always
+#spring.datasource.continueOnError=true
```

Comment:
Decided to disable this by default so we can see errors. Previously, this was true to ignore errors if a table was trying to get created that already existed or if data was duplicated.

Now, I have written in the `schema.sql` file a table drop and create if exists clause that should not error out.

In other words, if we get an error, we should investigate that problem instead of ignoring it.